locale/de/LC_MESSAGES/clipboard-history@alexsaveau.dev.po

```diff
@@ -145,19 +145,18 @@ msgid "Both"
 msgstr "Beides"
 
 #: prefs.js:329
+#, fuzzy
 msgid "Neither"
 msgstr "Nichts"
```

Comment:
"Neither" sounds like there are two or more options the user can choose from. In what context is this string used?

Reply:
https://github.com/SUPERCILEX/gnome-clipboard-history/blob/ba1a28be1a0c7fb7d40ad4d873eaab5a64cc3256/prefs.js#L329

It's for deciding what to show in the top bar: the icon, copied text, both those things, or neither of them (so nothing).